joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/Main.scala

```diff
@@ -32,7 +34,16 @@ private object Frontend {
         .action((path, c) => c.copy(inferenceJarPaths = c.inferenceJarPaths + path)),
       opt[Unit]("fetch-dependencies")
         .text("attempt to fetch dependencies jars for extra type information")
-        .action((_, c) => c.copy(fetchDependencies = true))
+        .action((_, c) => c.copy(fetchDependencies = true)),
+      opt[String]("feature-set-version")
```

Comment:
This seems to be unrelated to `lombok` as it is only used as input to JavaParser. Please put that into a different PR.